solutions/observability/apm/apm-server/use-internal-collection-to-send-monitoring-data.md

```diff
@@ -80,7 +80,7 @@ You specify these settings in the X-Pack monitoring section of the `apm-server.y
 
 ### `monitoring.enabled` [_monitoring_enabled]
 
-The `monitoring.enabled` config is a boolean setting to enable or disable {{monitoring}}. If set to `true`, monitoring is enabled.
+The `monitoring.enabled` config is a boolean setting to enable or disable X-Pack monitoring. If set to `true`, monitoring is enabled.
```

Comment:
I don't think "X-pack monitoring" is a thing anymore so it's likely better to use generic "monitoring features" terminology.
```suggestion
The `monitoring.enabled` config is a boolean setting to enable or disable {{monitor-features}}. If set to `true`, monitoring is enabled.
```